src/main/java/org/capstone/job_fair/controllers/account/AccountController.java

```diff
@@ -96,4 +106,21 @@
         }
     }
 
+    @PostMapping(ApiEndPoint.Account.PICTURE_PROFILE_ENDPOINT)
+    public ResponseEntity<?> uploadProfilePicture(@RequestParam("file") MultipartFile file) {
+        AccountDTO accountDTO;
+        try {
+            byte[] image = ImageUtil.ConvertImage(file, DataConstraint.Account.IMAGE_TYPE, DataConstraint.Account.WIDTH_FACTOR, DataConstraint.Account.HEIGHT_FACTOR);
+            String pictureProfileFolder = AWSConstant.PICTURE_PROFILE_FOLDER;
+            accountDTO = accountService.updateProfilePicture(pictureProfileFolder);
+            fileStorageService.store(image, pictureProfileFolder + "/" + accountDTO.getId()).exceptionally(throwable -> {
+                throwable.printStackTrace();
```

Comment:
Add @Slf4j to Controller class and replace throwable.printStackTrace(); with log.error(thowable.getMessage())